pkg/auth/auth.go

```diff
@@ -369,21 +369,21 @@ func (a *Authenticator) CallbackFunc(fn func(loginInfo LoginJSON, successURL str
 		}
 
 		if code == "" {
-			klog.Info("missing auth code in query param")
+			klog.Error("missing auth code in query param")
 			a.redirectAuthError(w, errorMissingCode)
 			return
 		}
 
 		if urlState != cookieState.Value {
-			klog.Error("State in url does not match State cookie")
+			klog.Error("state in url does not match State cookie")
```

Comment:
This one we should log at a higher level. I don't think we ever expect to see this.

Reply:
> I don't think we ever expect to see this.

in that case there should not be any log level set, since its not expected, right?